src/MSBuild/MSBuild.csproj

```diff
@@ -247,6 +240,14 @@
     <ProjectReference Include="..\Tasks\Microsoft.Build.Tasks.csproj" />
   </ItemGroup>
 
+  <Target Name="CopyXsds"
+          BeforeTargets="AfterBuild">
+    <Copy SourceFiles="@(XsdsForVS)"
+          DestinationFiles="@(XsdsForVS->'$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'xsd'))%(RecursiveDir)%(Filename)%(Extension)')"
```

Comment:
Why does this work? I thought you didn't assign it metadata.

Reply:
I don't understand the question. Can you rephrase? Are you asking about the [well-known metadata](https://docs.microsoft.com/visualstudio/msbuild/msbuild-well-known-item-metadata?view=vs-2019)?

Reply:
I'd forgotten about those. Thanks!